src/rock/models/finances/resolver.js

```diff
@@ -29,43 +29,28 @@ export default {
         id,
       });
     },
-    transactions: (
-      _,
-      { people, start, end, limit, cache, skip },
-      { models, person },
-    ) => {
+    transactions: (_, { people, start, end, limit, cache, skip }, { models, person }) => {
       if (!person) return null;
-      // NOTE: Not too sure what this query is trying to do
-      // it may be trying to put together transactions
-      // from all available aliases on a given user. In Holtzman
-      // this is likely broken also as the `people` object is hard-wired to
-      // a blank array (and person was returning inconsistently before also).
-      // if (person.GivingGroupId) {
-      //   console.log({ GivingGroupId: person.GivingGroupId, start, end, limit, skip, person });
-      //   return models.Transaction.findByGivingGroup(
-      //     {
-      //       id: person.GivingGroupId,
-      //       include: people,
-      //       start,
-      //       end,
-      //     },
-      //     { limit, offset: skip },
-      //     { cache },
-      //   );
-      // }
-
-      // console.log({ limit, skip, cache });
+      if (person.GivingGroupId) {
+        return models.Transaction.findByGivingGroup(
+          {
+            id: person.GivingGroupId,
+            include: people,
```

Comment:
The `people` parameter is never passed from the client nor assumed from the server. I think there may be a missing piece here but I need an account that has a `GivingGroupId` to be able to test this fully. @IsaacHardy can you help me with this?

Reply:
It looks like in my testing just now, the `people` array is indeed empty. But I'm guessing it's pulling the transactions based on my GivingGroupId. When I try the transactions query in GraphiQL I set the HTTP Header to use my auth token and left the `people` array empty and it pulled my transactions. @Lepozepo I messaged you what I think your giving group id might be...but I'm not sure what account your using for your testing.